src/mixs/schema/mixs.yaml

```diff
@@ -3813,12 +3813,12 @@ slots:
     required: true
   abs_air_humidity:
     annotations:
-      Preferred_unit: gram per gram, kilogram per kilogram, kilogram, pound
-    description: Actual mass of water vapor - mh20 - present in the air water vapor
-      mixture
+      Preferred_unit: gram per gram, kilogram per kilogram, kilogram, pound, gram per cubic meter, kilogram per cubic meter
```

Comment:
From a comment from @turbomam  & @only1chunts , I'm not sure gram per gram should be allowed?
I think we should remove gram per gram, kilogram per kilogram, kilogram, pound,
and add percent. 
Thoughts?

Reply:
https://github.com/GenomicsStandardsConsortium/mixs/issues/665#issuecomment-2809813613

Reply:
well i'm no expert (understatment! I've never meadsured it), but using google it would appear that the example given of "9 gram per gram" is equivelent to a relative humidity of approximately 63728.32 %, which seem unlikely to be possible to me! even 1 gram per gram is 5000%, so I think realistically people will be using "gram per kilogram" rather than grams per gram or kg per kg. The examples I've seen on internet suggest grams / cubic meter is more common way to express it.